src/v/cluster/controller.json

```diff
@@ -21,6 +21,11 @@
             "input_type": "create_topics_request",
             "output_type": "create_topics_reply"
         },
+        {
```

Comment:
Please place the `Fixes 3034` in the body of the commit and describe whats actually going on in the commit header message. You can take a look at current open PRs for the common patterns we use when making commit messages.